### ISSUE TITLE
[Input][SuggestionItem]: Suggestion items are not keyboard accessible on smaller screens.

### ISSUE THREAD

**Issue by bonnieso** [SAP/ui5-webcomponents#11183]
### Bug Description

Users on small screens can only bring up the suggestion items list by clicking on the input element. When a keyboard user types on a small screen, the suggestion items list don't show up.

### Affected Component

Input

### Expected Behaviour

It is expected that the keyboard user should be able to bring up the suggestion items list on small screens.

### Isolated Example

https://tinyurl.com/2ypt472a

### Steps to Reproduce

1. Open the example in a browser window that is smaller than 600px.
2. Using your keyboard, tab to the input field and type something.
3. Observe that the suggestion items list does not open.

### Priority

Medium

### UI5 Web Components Version

2.8.0

### Browser

Chrome

### Organization

SAP SuccessFactors

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by s-todorova** [comment]
Hello @bonnieso,

I am unable to reproduce the issue no matter how small i make the browser window. Could you provide more detailed instructions on how you are able to reproduce the problem - is 600px width or height wise? 

I tested with a 310x233 actual area screen and the suggestions show up:

![Image](https://github.com/user-attachments/assets/d5ffc09b-87ac-480a-afc2-ee9017213c1a)

Best regards,
Siyana

**Comment by bonnieso** [comment]
Hi @s-todorova ,

If you open Chrome developer tools and click the "Toggle device toolbar" icon on the top left and set the screen dimensions to a smaller width, you should be see the issue.

Best,
Bonnie

**Comment by s-todorova** [comment]
Hi @bonnieso,

Even with this approach I am unable to reproduce the issue:

![Image](https://github.com/user-attachments/assets/a3db1701-7173-466a-ba89-05730ed11f72)

If you can provide any more details about how to reproduce it, please do, otherwise we will have to close this as non-reproducible.

Best,
Siyana
